docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: "3.8"
+
+services:
+  todo_app:
+    container_name: todo_app
+#    build:
+#      context: .
+#      dockerfile: Dockerfile
+    image: todo-app:dev
+    env_file:
+      - .env
+    ports:
+    - 5000:5000
+    volumes:
+    - type: bind
+      source: ./todo_app
+      target: /app/todo_app
+      read_only: true
+  todo_app_tests:
+    container_name: todo_app_tests
+    image: todo-app:test
+    volumes:
+    - type: bind
+      source: ./todo_app
+      target: /app/todo_app
+      read_only: true
```

Comment:
This isn't strictly necessary to run the development container and in fact will break if the test image hasn't been pre-built:
```suggestion
```